Kudu.FunctionalTests/App.config

```diff
@@ -3,6 +3,10 @@
     <appSettings>
         <!-- Uncomment out this line to reuse the same site for all functional tests, which runs much faster -->
         <add key="SiteReusedForAllTests" value="TestRunnerSite" />
+        <!-- Set this to true to run the tests against Linux Kudu instead of Windows Kudu -->
+        <add key="RunningAgainstLinuxKudu" value="false" />

```

Comment:
Can we refer to it as KuduLite or something else? It is currently used just for App Service Linux but it can run on Windows and we might be used in other contexts

Reply:
For running tests against any arbitrary Kudu endpoint, `CustomKuduUrl` can be used. The `RunningAgainstLinuxKudu` property is for some 'Linux' specific checks, not exactly 'KuduLite' checks. I can rename it to `RunningAgainstKuduLite` but this check is slightly different than that.